packages/database/src/index.ts

```
@@ -1 +1,11 @@
+import { PrismaClient } from "@prisma/client";
+
+declare global {
+  var prisma: PrismaClient | undefined;
+}
+
+export const prisma = global.prisma || new PrismaClient();
```

Biome

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)